app/assets/stylesheets/mobile/mobile.scss

```diff
@@ -176,7 +176,7 @@ footer {
     border-radius: 3px 3px 0 0;
 
     border: {
-      bottom: 1px solid #ccc;
+      bottom: 1px solid $border-medium-grey;
```

Comment:
I love cleanups, so I'll do it sometime (if nobody else does it), but I'd rather not do it now as drive-by for a theme. ;)